docs/faq/events.md

```diff
@@ -28,3 +30,187 @@ There are lots of times when words just aren't enough to really get clarity on a
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
+
+## Event listening
+
+When preparing to listen for events in your code, do the least work possible. Not only because this allows you to do less work now, but it allows you also to do less work later, and most importantly it allows the browser to do less work, which means a more performant application or component. The least work possible changes per context, but in important to remember no matter where you are adding event listeners.
+
+### Inside of your elements
+
+When adding event listening inside of your components, assuming you are using `lit-html` (or any other declarative renderer with event binding syntax) use the tools supplied therein to [add event listeners](https://lit-html.polymer-project.org/guide/writing-templates#add-event-listeners). The reasoning behind this is that for any listener added by `lit-html`, the same listener will be removed by `lit-html` when no longer needed; no ghost listeners! That means whether the DOM and its associated listener is available for the entire lifecycle of your application, or just for a small part of that, the least amount of work will be done to ensure the event is handled appropriately. See the `click` listener in the following code. It _only_ exits while `this.open` is `true`, which leaves the browser listening to fewer things and your code less likely to trigger handlers errantly.
+
+```js
+render() {
+    return html`
+        ${this.open
+            ? html`<button @click=${this.handleClose}>Close</button>
+            : html``
+        }
+    `;
+}
+```
+
+A side benefit of relying on the `@event` syntax in `LitElement` (if you use it) is that all events handled therein are automatically bound to `this` which will prevent the need to do so manually: no `this.handleClose = this.handleClose.bind(this);` and, no `handleClose = (event) => {}` in the case above.
+
+### On your elements
+
+If you are adding a listener to your element, from inside your element, you should add that listener in the `constructor`. The browser does a great job of garbage collecting that listener when the element is removed from the DOM, so there is no reason to do any work to manage that yourself. See the following example where the element listens for a `done` event.
+
+```js
+constructor() {
+    super();
+    this.handleDone = this.handleDone.bind(this);
+    this.addEventListener('done', this.handleDone);
```

Comment:
if you don't need a link to the original thing then you can do
```js
this.addEventListener('done', this.handleDone.bind(this));
```
directly

Reply:
Good call. This will also strengthen the note on keeping a reference below.